question.html

```diff
@@ -166,15 +144,12 @@ <h4 class="media-heading">
 						<!-- Replies Body -->
 						<div class="media-body">
 							<!--  Replies Heading-->
-							<h4 class="media-heading">
-								<span class="badge progress-bar-warning">{{-reply.order}}</span>
-								<span ng-class="{completed: todo.completed}">Reply to: {{todo.head}}</span>
```

Comment:
Removing this title since it contains no information and we never agreed to display it.
This should have never been merged into master.